cmd/github-ldap-user-group-creator/main.go

```diff
@@ -290,6 +302,51 @@ type GroupClusters struct {
 
 var githubRobotIds = sets.New[string]("RH-Cachito", "openshift-bot", "openshift-ci-robot", "openshift-merge-robot")
 
+func deleteUsersNotInRover(ctx context.Context, clients map[string]ctrlruntimeclient.Client,
+	kerberosIDs sets.Set[string], dryRun bool) error {
+
+	var errs []error
+	for cluster, client := range clients {
+		usersNotInRover, err := getUsersWithoutKerberosID(ctx, client, cluster, kerberosIDs)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("failed to get users on cluster %s: %w", cluster, err))
+		}
+		for user, hisIdentites := range usersNotInRover {
+			logrus.Infoln("Deleting user", user, "with his identites on cluster", cluster)
+			if dryRun {
+				continue
+			}
+			for _, identity := range hisIdentites {
+				if err := client.Delete(ctx, &userv1.Identity{ObjectMeta: metav1.ObjectMeta{Name: identity}}); err != nil && !errors.IsNotFound(err) {
```

Comment:
According to the doc, we should delete the user first, then the identies.
https://docs.openshift.com/container-platform/3.11/admin_guide/manage_users.html#managing-users-deleting-a-user

Reply:
Maybe add a log when delete is completed successfully?